.gitignore

```diff
@@ -6,3 +6,4 @@
 venv/
 .pytest_cache/
 **__pycache__/
+db.sqlite3
```

Comment:
Remove the extra space at the end of the `db.sqlite3` entry to ensure it is correctly ignored by Git.